feature_appview/src/main/java/cm/aptoide/pt/feature_appview/data/AptoideAppViewRepository.kt

```diff
@@ -84,16 +85,26 @@ class AptoideAppViewRepository @Inject constructor(
     return flow {
       val relatedContentResponse = remoteAppViewRepository.getRelatedContent(packageName)
       if (relatedContentResponse.isSuccessful) {
-        relatedContentResponse.body()?.datalist?.list?.let {
-          emit(RelatedContentResult.Success(it.map { relatedCardJson -> relatedCardJson.toDomainModel() }))
+        relatedContentResponse.body()?.datalist?.list?.map { relatedCard ->
+          reactionsRepository.getTotalReactions(relatedCard.id)
```

Comment:
I already refactored reactions a bit, so you can use that

Reply:
I can only use that after your PR gets merged or already in code?